### PR TITLE
Upgrade Python MQTT clients to MQTTv5 

### PR DESCRIPTION
**Pull request by karol-brejna-i**
- Update libraries (fast_mqtt)
- Allow for MQTT protocol version configuration with env variable (choose v5 as the default value)

### CODE REVIEW

File: visualization-app/app/main.py [hunk 1]
```diff
@@ -1,46 +1,30 @@
...
 import json
-import random
-import requests
-import ssl
 import time
+import uuid
 
-from gmqtt.mqtt.constants import MQTTv311
-
+# TODO random probably not used
```
Comment (karol-brejna-i) [inline visualization-app/app/main.py:6]: @pgrabusz could you take a look at this one?

**Comment by karol-brejna-i** [comment]
If the external method for MQTT client initialization works, I'll implement it in other python web services.

**Review by pgrabusz — APPROVED** [review]

File: visualization-app/app/utils.py [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+        context.load_verify_locations(config.MQTT_BROKER_CERT_FILE)
+    protocol_version = MQTTv311 if config.MQTT_PROTOCOL_VERSION == 'MQTTv311' else MQTTv50
+    mqtt_config = MQTTConfig(
+        host=config.MQTT_HOST,
+        port=config.MQTT_PORT,
+        username=config.MQTT_USERNAME,
+        password=config.MQTT_PASSWORD,
+        version=protocol_version,
+        ssl=context)
+    mqtt = FastMQTT(config=mqtt_config)
```
Comment (karol-brejna-i) [inline visualization-app/app/utils.py]: Set client id on init.

Comment (karol-brejna-i) [reply visualization-app/app/utils.py]: resolved

**Review by karol-brejna-i — COMMENTED** [review]
Let's merge

**Pull request merged**